### PR TITLE
PL-99: Add Triptional methods for examining the value

### PR DESCRIPTION
**Pull request by effiban**
To support a use-case that has already come up in Kenshoo search - adding three methods for checking the value of a `Triptional`.
`filter()` already exists in `Optional`.
The other two exist in the Scala `Option`. We chose different names:
- `equalsValue()` instead of `contains()`
- `matches()` instead of `exists()`

### CODE REVIEW

**Comment by effiban** [comment]
@amirz

File: main/src/main/java/com/kenshoo/pl/entity/Triptional.java [hunk 1]
```diff
@@ -314,6 +311,50 @@ public void ifNotNull(final Consumer<? super T> consumer) {
...
+    /**
+     * Return {@code true} if there is a value present (possibly {@code null), and it equals the input;
+     * otherwise {@code false}
+     *
+     * @param value a value to compare to this value
+     * @return {@code true} if there is a value present, and it equals the input; otherwise {@code false}
+     */
+    public boolean contains(final T value) {
+        return isPresent() && Objects.equals(this.value, value);
+    }
```
Comment (effiban) [inline main/src/main/java/com/kenshoo/pl/entity/Triptional.java:323]: For completeness - there should also be an option to specify an equality function like `equals()` has.
Leaving that for a PR later on

**Comment by effiban** [comment]
@galkoren - comments fixed (rename `contains()` to `equalsValue()` and add test for both `null`)

**Review by galkoren — APPROVED** [review]
